### PR TITLE
Read encoders method

### PR DESCRIPTION
**Pull request by bodiddles**
Here's the code that supports reading encoders in the form of get_motor_distance in Robot.py. As you can probably see, it's probably not the most efficient way of implementing this as it constantly updates memcache with encoder values but it definitely works.

### CODE REVIEW

**Comment by matthew-zhao** [comment]
Has this code been tested? Can I merge it

**Comment by bodiddles** [comment]
I did test it and it works but I didn't try stress testing it yet

**Comment by nikitakit** [comment]
Don't merge until past merge conflicts are fized

File: runtime/runtime.py [hunk 1]
```diff
@@ -20,7 +20,13 @@
 mc.set('drive_distance', [])
 mc.set('metal_detector_calibrate', [False,False])
 mc.set('toggle_light', None)
+<<<<<<< HEAD
+mc.set('game', {'autonomous': False, 'enabled': False})
+mc.set("spec_pid", [])
+mc.set("encoder_distance", {})
+=======
 mc.set('game', {'autonomous': False, 'enabled': True})
+>>>>>>> 7bfaff1495de5a78e92d485eff5f28985836e319
```
Comment (nikitakit) [inline runtime/runtime.py]: This section needs to be resolved. Please be sure to resolve it correctly

**Comment by matthew-zhao** [comment]
Bob can you fix this tonight? I can also just pull your code and fix that conflict.

**Comment by bodiddles** [comment]
Fixed. Will make new pull request.

**Pull request closed without merging**